### PR TITLE
Feature: CLI

### PR DESCRIPTION
**Pull request by h-alice**
We implemented a simple CLI interface.
```bash
imagetool [-f Config path]... [input images]...
-f Add Image processing config, can assign multiple config.
--help Display help.
```

### CODE REVIEW

**Review by boris-lok — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,8 @@ vendor
 .vscode
 
 # Local test files.
-local_test_files
\ No newline at end of file
+local_test_files
+
+# Artifacts generated by the build process.
+imagetools
+imagetools.exe
```
Comment (NatLee) [inline .gitignore:13]: Wow, exe!

File: main_interactive.go
Comment (NatLee) [inline main_interactive.go:1]: rename uwu

File: main.go [hunk 2]
```diff
@@ -1,149 +1,174 @@
...
+		Action: func(c *cli.Context) error {
+
+			// Placeholder for input config file paths.
+			loaded_configs := make([]config.ProfileRoot, 0) // Placeholder for loaded configs.
+
+			// Placeholder for config root.
+			config_root := config.ProfileRoot{}
+
+			// First, check if any argument is specified.
+			if c.NArg() == 0 {
```
Comment (NatLee) [inline main.go:99]: Parser is a considerable annoyance for many developers.

**Review by NatLee — APPROVED** [review]
Ready to go!

File: main.go (hunk 2 above)
Comment (h-alice) [reply main.go:99]: So I think it's a good practice to make it an isolated file. : D

File: main_interactive.go
Comment (h-alice) [reply main_interactive.go:1]: uwu

File: .gitignore (hunk 1 above)
Comment (h-alice) [reply .gitignore:13]: I forgot desktop.ini 🥲

**Comment by h-alice** [comment]
Let's GOOO

**Pull request merged**